tests/test_utils.py

```diff
@@ -62,3 +70,42 @@ async def test_get_template(notification_template, async_client, token):
     assert template3.name == "notification"
     assert template3.template_text == template2.template_text
     await async_client.delete("/templates/1", headers={"Authorization": f"Bearer {token}"})  # cleanup
+
+
+@pytest.mark.asyncio
+async def test_notify_template():
+    template = await utils.get_notify_template(MockStore(), "invoice")
+    assert template.startswith("New order from")
+
+
+@pytest.mark.asyncio
+async def test_product_template(async_client, token):
+    qty = 10
+    resp = await async_client.post(
```

Comment:
Why is this post request needed? What does it do in this case?

Reply:
good to learn. I thought we need a unique template for each category